src/dwalk/dwalk.c

```diff
@@ -238,7 +238,8 @@ static void print_usage(void)
     printf("\n");
     printf("Options:\n");
     printf("  -i, --input <file>                      - read list from file\n");
-    printf("  -o, --output <file>                     - write processed list to file\n");
+    printf("  -o, --output <file>                     - write processed list to FILE in binary format\n");
```

Comment:
To be consistent, let's use "file" instead of "FILE" in the print_usage.  I do prefer FILE, actually, since that sets it off more that it's a placeholder.  However, then we should update the rest of the print_usage to use a style of all caps.

What do others prefer?

Reply:
I think we are more consistent in the man pages. I'll revert to "file" here and leave it as a future exercise to fix-up the usage documentation.